### DISCUSSION TITLE
高セキュリティ環境での使用について

### DISCUSSION THREAD

**Discussion by apro0666** [Yowkees/keyball#376] [General]
高セキュリティ下のKeyballの利用について確認させてください。

昨日アサインしたプロジェクトのセキュリティが高く、支給されたPCにIphionなど携帯デバイスを接続した場合はセキュリティーアラートが発生すると言われております。
※おそらく登録外のコンピュータが例外的接続をしたとしてセキュリティソフトのルールに抵触するものと思われる。

USBケーブルケーブル経由の情報がPC側にどう伝わるかという問題と思われますが、KeyBallを接続した場合下記どちらになるのでしょうか。
・PC側（セキュリティソフト）からは普通のキーボードとして認識される。
　　＝＞セキュリテエラーにならない。
・PC側（セキュリティソフト）からはPromicro（コンピュータ）が接続されたと認識される。
　　＝＞セキュリティエラーになる。
　
ご教示のほどお願いいたします。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

OSやセキュリティー関連について知識がなく、おそらくですが、
後者の「PC側（セキュリティソフト）からはPromicro（コンピュータ）が接続されたと認識される」と思われます。
高いセキュリティー環境では使用しないことをお勧めします。

**Reply by apro0666** [reply]
ご返信ありがとうございます。
やはりそういった情報はないですよね。
今回は利用を見送ろうと思います。
